presto-main/src/main/java/io/prestosql/testing/TestingEventListenerManager.java

```diff
@@ -30,6 +31,11 @@
 public class TestingEventListenerManager
         extends EventListenerManager
 {
+    public static TestingEventListenerManager emptyConfigManager()
```

Comment:
```suggestion
    public static TestingEventListenerManager testingEventListenerManager()
```
?

`emptyConfigManager` sounds like a manager of empty config o_O

Reply:
But that's exactly the case, it's an EventLitenerManager with empty config of event listeners, so with no event listeners configured. Maybe noListenersConfiguredConfigManager()? As testingEventListenerManager is too broad as it's not giving any hint of its state (which is important for the tests).

Reply:
`emptyEventListenerManager`?

Reply:
hm you are right, it's better :)